fs/filebench.py

```diff
@@ -52,9 +52,9 @@ def setUp(self):
                                    'filebench/releases/download/1.5-alpha3/'
                                    '%s.tar.gz' % name_version)
 
-        archive.extract(tarball, self.srcdir)
+        archive.extract(tarball, self.wordir)
```

Comment:
In this test, `self.workdir` is used already for another purpose. Please take care of having unique directories.

Reply:
True.  The test actually PASSes like this, because the installation will just create the directories and files on top of the source directory, but you're right, it makes sense to use different directories.